app/controllers/api/apps_controller.rb

```diff
@@ -3,6 +3,15 @@ class Api::AppsController < Api::BaseController
 
```

Comment:
just return 404, and you also have to check if app group is active here
I think it's better if you have another method in BaritoApp model called `#available?` it checks for app.active? and infra active?

Reply:
Nice suggestion. Will do it